docs/release-notes/release-notes-0.7.1.md

```diff
@@ -0,0 +1,117 @@
+Joinmarket-clientserver 0.7.1:
+=================
+
+<https://github.com/joinmarket-org/joinmarket-clientserver/releases/tag/v0.7.1>
+
+This release requires Python 3.6+ - see "Upgrading" section in the 0.7.0 release notes for more information if needed.
+
+This implementation includes support for **receiving BIP78 (payjoin) payments** (0.7.0 introduced sending them), in Qt GUI or on command line, using either current P2SH segwit wallets ('3' addresses) or native bech32 wallets.
+To support this, Joinmarket spawns a Tor onion service ("hidden service") temporarily for payment receipt, for maximum privacy.
+
+There are a few other fixes and improvements as listed below.
+
+Please report bugs using the issue tracker at github:
+
+<https://github.com/joinmarket-org/joinmarket-clientserver/issues>
+
+Upgrading 
+=========
+
+To upgrade:
+
+(If you are upgrading from a version pre-0.6.2 then please see the section "Move user data to home directory" in [the 0.6.2 release notes](https://github.com/JoinMarket-Org/joinmarket-clientserver/blob/master/docs/release-notes/release-notes-0.6.2.md), and follow the instructions there (except the `commitmentlist` file - that can actually be left alone, the previous release notes were wrong on this point).)
+
+(If you are upgrading from a version pre-0.7.0 please read the "Upgrading" section in [the 0.7.0 release notes](https://github.com/JoinMarket-Org/joinmarket-clientserver/blob/master/docs/release-notes/release-notes-0.6.2.md).)
+
+First run the `install.sh` script as mentioned in the README. When prompted to overwrite the directory `jmvenv`, accept.
+
+Note that `./install.sh -?` will show the options for installation.
+
+Notable changes
+===============
+
+### BIP 78 (Payjoin) Receiver
+
+Joinmarket should now be able to send *and* receive payjoins.
+
+For help in usage, see the [documentation](../PAYJOIN.md). There is also a simple video demonstration [here](https://video.autizmo.xyz/videos/watch/7081ae10-dce0-491e-9717-389ccc3aad0d).
+This completes Joinmarket's implementation of [BIP 78 Payjoin](https://github.com/bitcoin/bips/blob/master/bip-0078.mediawiki). The receiver uses a Tor hidden service, so Tor must be running on the receiver's machine (see the docs for some details on this). There have been some limited compatibility tests with other wallets, but more testing in the field will be helpful (please report on this github repo if you find issues in using).
```

Comment:
```suggestion
This completes Joinmarket's implementation of [BIP 78 Payjoin](https://github.com/bitcoin/bips/blob/master/bip-0078.mediawiki). The receiver uses a Tor onion service, so Tor must be running on the receiver's machine (see the docs for some details on this). There have been some limited compatibility tests with other wallets, but more testing in the field will be helpful (please report on this github repo if you find issues in using).
```

Reply:
Done in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/701/commits/01aad72ef0a5413240fa8825ee75833cba994e11